app/tsconfig.json

```diff
@@ -22,7 +22,8 @@
       "@layouts/*": ["layouts/*"],
       "@pages/*": ["pages/*"],
       "@stories/*": ["stories/*"],
-      "@styles": ["styles/*"]
+      "@styles": ["styles/*"],
```

Comment:
Should it have the `/*` at the end or not?
Check which is the correct one and apply to all.